src/main/kotlin/org/rust/ide/annotator/fixes/ConvertToRefTyFix.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator.fixes
+
+import com.intellij.psi.PsiElement
+import org.rust.lang.core.types.ty.Ty
+
+/**
+ * For the given `expr` converts it to the reference type with `as_ref()` method.
+ */
+class ConvertToRefTyFix(expr: PsiElement, ty: Ty) : ConvertToTyUsingTraitFix(expr, ty, "AsRef", "as_ref")
```

Comment:
I think it's ok to move all these single-line classes to `ConvertToTyUsingTraitFix.kt` file

Reply:
Done